flowdb/testdata/bin/run_qa_checks.py

```diff
@@ -0,0 +1,137 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from dataclasses import asdict, dataclass
+from datetime import date, datetime
+from itertools import product
+from pathlib import Path
+from typing import List
+from jinja2 import Environment, PackageLoader, Template
+from sqlalchemy import create_engine
+from sqlalchemy.engine import Engine
+import os
+import argparse
+
+
+env = Environment(loader=PackageLoader("flowetl", "qa_checks/qa_checks"))
```

Comment:
Presumably we need to copy these into the container as well?

Reply:
Just sticking them into the `dockerignore` should grab it, right?